core/src/main/java/hudson/model/Slave.java

```diff
@@ -133,10 +133,7 @@
     private /*almost final*/ DescribableList<NodeProperty<?>,NodePropertyDescriptor> nodeProperties = 
                                     new DescribableList<NodeProperty<?>,NodePropertyDescriptor>(Jenkins.getInstance().getNodesObject());
 
-    /**
-     * Lazily computed set of labels from {@link #label}.
-     */
-    private transient volatile Set<Label> labels;
```

Comment:
Yes, I understand it, but I think that it is consequence of a direct binding of the plugins with the internal implementation details. I agree on applying these kind of changes in the 2.0 version.

Removing unused imports or variables is part of the coding style, but also any of the changes in the Java source file as well. However, in contrast with the field and method declarations, it can't break plugins.